src/main/java/core/basesyntax/db/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.db;
+
+public class FruitTransaction {
+    private String fruit;
+    private int quantity;
+    private String type;
+
+    public FruitTransaction(String fruit, int quantity) {
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public FruitTransaction(String type, String fruit, int quantity) {
+        this.type = type;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType() {
+        this.type = type;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+}
```

Comment:
The class `FruitTransaction` should be focused on a single representation of a transaction. Currently, there is a constructor on lines 8-11 that does not include the transaction type which is a critical attribute of a transaction. This could lead to incomplete `FruitTransaction` objects. Consider removing this constructor to enforce the creation of complete transaction objects.